go/pkg/cas/upload.go

```diff
@@ -395,7 +395,13 @@ func (c *Client) Upload(ctx context.Context, opt UploadOptions, inputC <-chan *U
 		}
 	})
 
-	return &UploadResult{Stats: u.stats, u: u}, errors.WithStack(eg.Wait())
+	err := eg.Wait()
+	if err != nil {
+		// Caller may not unwrap stack frame of github.com/pkg/error, so log error here.
+		log.Errorf("upload err %+v", err)
```

Comment:
Maybe a more descriptive message? If only for a client grepping for error code to find out easier :) eg "failed to upload at least one blob: %+v"

Reply:
done